app/inbox/admin.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+"""
+
```

Comment:
can we please add an admin for the notifications object?

Reply:
There is an admin area for notifications, is that what you mean?